woocommerce/woocommerce
# CYS - Fix spacing in patterns

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds margins to the following patterns to align with Dotcom patterns:
- Featured Category Cover Image (Intro category)
- Hero Product Split (Intro category)
- Banner (Featured selling category)
- Hero Product 3 Split (Featured selling category)
- Hero Product Chessboard (Featured selling category)
- Content right with image left (About category)



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com` and uncheck `Allow usage of WooCommerce to be tracked`
2. Now recheck it and save.
3. Go to `wp-admin/tools.php?page=action-scheduler`, search for the `fetch_patterns` action, and ensure it was scheduled.
4. Click on `Run` under the action name if it is still pending and wait until it is completed.
5. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`, click on `Start designing` or `Customize your theme`.
6. In the assembler, click on `Design your homepage`.
7. For each of the following patterns check they have margins top & bottom:
- Featured Category Cover Image (Intro category)
- Hero Product Split (Intro category)
- Banner (Featured selling category)
- Hero Product 3 Split (Featured selling category)
- Hero Product Chessboard (Featured selling category)
- Content right with image left (About category)



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Improve margins for CYS core patterns.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
